README.md

```diff
@@ -1,11 +1,153 @@
-___
-**NOTE: Documentation is outdated, needs updating!**
-___
+# XayNet: federated learning made private, performant, and ubiquitous
 
-# XAIN Federated Learning
+###### tags: Xayn, Federated Learning, Privacy
 
 This is the main source code repository for [xain-fl](https://www.xain.io/).
 
-To get started, take a look at the documentation: https://xain-fl.readthedocs.io/en/latest
+---
 
-For developers, see [CONTRIBUTING.md](./CONTRIBUTING.md)
+> ### Want a framework that supports federated learning on the edge, in desktop browsers, integrates well with mobile apps, is performant, and preserves privacy? Welcome to XayNet, written entirely in Rust!
+
+## Making federated learning easy for developers
+Frameworks for machine learning - including those expressly for federated learning - exist already. These frameworks typically require the use of specific machine learning technology - for example tensorflow - or facilitate federated learning of cross-silo use cases - for example in collaborative learning across a limited number of hospitals.
+We want to give developers more freedom of choice and abilities in the creation of federated learning software. By doing this, we hope to also increase the pace and scope of adoption of federated learning in practice.
+Concretely, we provide developers with:
+- [**My AI tools**] the flexibility to use the machine-learning frameworks and tools of their choice,
+- [**My app dev tools**] the ability to integrate federated learning into apps written in Dart, Flutter or other languages of choice,
+- [**“Federated learning” everywhere**] the ability to run federated learning everywhere - be it desktop browsers, smartphones or micro-controllers,
+- [**“Federated learning” inside**] a simple integration means of making an AI application ready for federated learning,
+- [**Privacy by design**] a communication protocol for federated learning that scales, is secure, and preserves the privacy of participating devices.
```

Comment:
i'd suggest to remove the braces, they are not rendered in markdown (i think they were initially added as Latex enumeration items). same in the next section.

i'd also suggest to have the same style for all enumerations, like periods instead of commas. i did the same when adjusting the text for the rust docs.

```suggestion
- **My AI tools** The flexibility to use the machine-learning frameworks and tools of their choice.
- **My app dev tools** The ability to integrate federated learning into apps written in Dart, Flutter or other languages of choice.
- **“Federated learning” everywhere** The ability to run federated learning everywhere - be it desktop browsers, smartphones or micro-controllers.
- **“Federated learning” inside** A simple integration means of making an AI application ready for federated learning.
- **Privacy by design** A communication protocol for federated learning that scales, is secure, and preserves the privacy of participating devices.
```

Reply:
Good point! Done!